lib/identity-provider.js

```diff
@@ -524,15 +531,77 @@ IdentityProvider.prototype.getGraph = function (uri, callback) {
     })
 }
 
-// Handle POST requests on account creation
-IdentityProvider.prototype.post = function (req, res, next) {
-  if (!req.body) {
+/**
+ * Sends a request to the OIDC Provider's Users API endpoint, to create a new
+ * user record with the provider.
+ * @method createOIDCUser
+ * @param oidcRpClient {OidcRpClient} Local/trusted OIDC client
+ * @param webId {String} WebID URL of the new user to be created
+ * @param options {Object} User options hashmap
+ * @param options.password {String} User's signin password. NOTE: Must be 8+
```

Comment:
this should be optional too, we can of course pass a setting flag that is `--auth-password` or similar, we shouldn't enforce password as auth schema